x/ecocredit/base/keeper/msg_send.go

```diff
@@ -22,169 +22,233 @@ func (k Keeper) Send(ctx context.Context, req *types.MsgSend) (*types.MsgSendRes
 	sender, _ := sdk.AccAddressFromBech32(req.Sender)
 	recipient, _ := sdk.AccAddressFromBech32(req.Recipient)
 
-	if sender.Equals(recipient) {
```

Comment:
Can you also remove NOTE from the description?

Reply:
https://github.com/regen-network/regen-ledger/pull/1680/commits/a2df9e4635287b6adb61c890b215fef72fe1c9e0